### ISSUE TITLE
minimal data

### ISSUE THREAD

**Issue by gbif-portal** [gbif/data-mobilization#332]
## minimal data

Region: china

Taxon: diversity of species

Comments: Table 1 Distribution of species and dominant families at different elevation gradients
Elevation gradient(m)	Family	Genus	Species	Brachytheciacea	Hypnaceae	Thuidiaceae
Ⅰ（970-1151）	5	10	14	4	1	2
Ⅱ（1152-1333）	6	11	16	8	2	2
Ⅲ（1334-1515）	8	13	21	9	5	2
Ⅳ（1516-1697）	7	12	20	4	3	2

Table 2 Vertical distribution of dominant bryophyte genera
Elevation gradient(m)	Brachythecium	Eurhynchium	Hypnum	Plagiomnium	Thuidium	Anomodon
Ⅰ（970-1151）	3	1	0	1	0	1
Ⅱ（1152-1333）	6	2	2	1	1	0
Ⅲ（1334-1515）	7	3	3	3	2	2
Ⅳ（1516-1697）	3	1	1	0	1	0

Table 3 Vertical distribution of dominant bryophyte species
Elevation gradient(m)	Eurohypnum leptothallum	Brachythecium salebrosum	Brachythecium pendulum	Brachythecium amnicolum	Hyophila involuta	Entodon prorepens	Brachythecium garovaglioides	Thuidium delicatulum	Brachythecium albicans	Eurhynchium longirameum
Ⅰ（970-1151）	33	0	6	5	14	0	5	0	0	10
Ⅱ（1152-1333）	14	1	15	0	7	2	8	0	5	0
Ⅲ（1334-1515）	17	21	1	12	0	5	0	12	0	1
Ⅳ（1516-1697）	4	2	0	0	0	20	0	0	0	0

Table 4 Vertical distribution of dominant bryophyte species
Elevation gradient(m)	Wefts	Turfs	Pendants	Mat	Tail
Ⅰ（970-1151）	74	13	3	0	0
Ⅱ（1152-1333）	79	11	0	0	0
Ⅲ（1334-1515）	79	3	1	6	1
Ⅳ（1516-1697）	63	27	0	0	0

Table 5 Patrick richness index of different altitude gradients
Elevation gradient(m)	Mean value	Standard deviation
Significant difference labeling

Ⅰ（970-1151）	22	1.20185	c
Ⅱ（1152-1333）	25	0.33333	b
Ⅲ（1334-1515）	33	1.1547	a
Ⅳ（1516-1697）	29	0.88192	b

Table 6 Shannon-Wiener diversity index of different altitude gradients
Elevation gradient(m)	Mean value	Standard deviation
Significant difference labeling

Ⅰ（970-1151）	2.13	0.01155	c
Ⅱ（1152-1333）	2.4	0.00882	a
Ⅲ（1334-1515）	2.53	0.00577	b
Ⅳ（1516-1697）	2.52	0.00577	a

Table 7 Pielou evenness index of different altitude gradients
Elevation gradient(m)	Mean value	Standard deviation
Significant difference labeling

Ⅰ（970-1151）	0.75	0.00577	c
Ⅱ（1152-1333）	0.87	0.00577	a
Ⅲ（1334-1515）	0.75	0.00577	c
Ⅳ（1516-1697）	0.81	0.00577	b

Table 8 Simpson dominance index of different altitude gradients
Elevation gradient(m)	Mean value	Standard deviation
Significant difference labeling

Ⅰ（970-1151）	0.18	0.00577	a
Ⅱ（1152-1333）	0.13	0.00577	d
Ⅲ（1334-1515）	0.09	0.00577	b
Ⅳ（1516-1697）	0.11	0.00577	c

**Comment by CecSve** [comment]
@thomasstjerne - did you submit this and should it remain in the repo?

**Comment by timrobertson100** [comment]
Should "suggest a dataset" not include the logged-in user as it posts?

**Comment by CecSve** [comment]
> Should "suggest a dataset" not include the logged-in user as it posts?

Yes, I believe so. I asked because Thomas assigned Tobias and I thought they might know why this issue was created. It was registered through the form though.
